docs/index.bs

```diff
@@ -1952,9 +1952,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     A <dfn id="dfn-name-to-cache-map">name to cache map</dfn> is an <a>ordered map</a> whose [=map/entry=] consists of a [=map/key=] (a string that represents the name of a [=request response list=]) and a [=map/value=] (a [=request response list=]).
 
-    Each [=/storage key=] has an associated [=name to cache map=].
-
-    The <dfn id="dfn-relevant-name-to-cache-map">relevant name to cache map</dfn> is the [=name to cache map=] associated with the [=/storage key=] [=obtain a storage key|obtained=] from [=this=]'s associated [=CacheStorage/global object=]'s [=environment settings object=].
+    The <dfn id="dfn-relevant-name-to-cache-map">relevant name to cache map</dfn> is a [=name to cache map=] obtained as the result of running [=obtain a local storage bottle map=] with [=this=]'s associated [=CacheStorage/global object=]'s [=environment settings object=] and "<code>caches</code>".
```

Comment:
nit: Isn't below better describes indirect relationship which we want to establish

The <dfn id="dfn-relevant-name-to-cache-map">relevant name to cache map</dfn> is the [=name to cache map=] associated with the result of running [=obtain a local storage bottle map=] with [=this=]'s associated [=CacheStorage/global object=]'s [=environment settings object=] and "<code>caches</code>".

Reply:
sure. updated. also mixed with domenic's comment.